### ISSUE TITLE
Feature Request: Block all users who have liked a specific tweet.

### ISSUE THREAD

**Issue by cjay567** [satsukitv/twitter-block-chain#43]
Sometimes on Twitter people will be targetted for harassment by someone tweeting an attack against that person. Since reporting tweets doesn't always work, a handy tool to have would be one that can block all users who have liked a certain tweet. I'd imagine it would work by opening up the list of people who have liked the tweet and then activating the blockchain like normal.

**Comment by glitterizer** [comment]
Yes, please! I would LOVE this feature too, for RTs as well in addition to likes.

I'm willing to pay/donate money to get it ASAP.
@satsukitv would that kind of negotiation be available?
